cryptoki-sys/build.rs

```diff
@@ -103,7 +103,9 @@ mod generate {
             .derive_debug(true)
             // Derive the `Default` trait for the generated structs where possible.
             .derive_default(true)
-            .parse_callbacks(Box::new(CargoCallbacks));
+            .parse_callbacks(Box::new(CargoCallbacks))
+            // Support function like macros
```

Comment:
Can you maybe add a link to the issue that was fixed with that or a bit more context :)?

Reply:
All set